libtcmu_common.h

```diff
@@ -75,6 +77,9 @@ int tcmu_emulate_read_capacity_16(uint64_t num_lbas, uint32_t block_size, uint8_
 int tcmu_emulate_mode_sense(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 int tcmu_emulate_mode_select(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 
+void tcmulib_track_aio_request_start(struct tcmu_device *dev);
+void tcmulib_track_aio_request_finish(struct tcmu_device *dev, int *is_idle);
```

Comment:
This is used in the current PR in the thread cleanup handler for io_work_queue thread (_untrack_in_flight_io). We have just a single worker thread atm, but if at all we move to >1 worker threads, this is useful (either way).